main.py

```diff
@@ -102,6 +108,9 @@ def main():
     if not os.path.exists(save_path):
         os.makedirs(save_path)
 
+    if args.seed_split:
```

Comment:
Assuming we specify the int type in argparse (see above), if the user give the seed 0, this will be false.

The less ambiguous way here is to specifically ask if the split is None

```python
if argsd.seed_plit is not None:
```